src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1678,6 +1678,23 @@ class TcgStatics {
     }
   }
 
+  static bringDown (Object delegate) {
+    delegate.onAttack {
+      def list = all.findAll{it!=delegate.self}.sort(false) {p1,p2 -> p1.remainingHP.value <=> p2.remainingHP.value} as PcsList
+      def tar = new PcsList()
+      int min = list.get(0).remainingHP.value
+      while (list.notEmpty && list.get(0).remainingHP.value==min) {
+        tar.add(list.remove(0))
+      }
+      //TODO: Heavily improve this selection, in case both players have tied Pokémon. Make it clearer to pick.
```

Comment:
What does this mean?

Reply:
For context, this was already in the bring down code from greninja, it wasn't added now.

Reply:
As you might be aware, the new pokemon selection system has been rolled out. It makes all possible selections glow on the board. So there can be no confusion.

Reply:
Ahh, great this is solved then! Can remove that comment 😅 

Will check on test to confirm tho

Reply:
Yup, works as a charm!